src/components/content-box/content-box.html

```diff
@@ -51,6 +51,315 @@ <h3 class="docs-block__header">Simple</h3>
 </section>
 <section class="docs-block">
     <aside class="docs-block__info">
+        <h3 class="docs-block__header">Simple with title</h3>
```

Comment:
Added various combinations of `__title` +  `__header` + `__content` + `__actions` to see space squashing and wrapping